lib/biran/configurinator.rb

```diff
@@ -103,11 +103,12 @@ def local_config_file_contents
       @local_config_contents = process_config_file(local_config_file)
     end
 
-    def get_secret_contents(app_config)
+    def get_secrets_content(app_config)
       secrets_file_contents = {}
       if File.exist? app_config[:secrets_file_path]
         secrets_file_contents = process_config_file app_config[:secrets_file_path]
       end
+      return app_config[:secrets].deep_merge! secrets_file_contents unless app_config[:secrets].nil?
```

Comment:
do you want to check just against `nil` or `""` too?

Reply:
Well, by default we set it to `{}`, hard coded in the config.rb file. So the object exists. Isn't nil enough at that point? How can you set it to "" at that point?

Reply:
I see. NVM, then.